spec/rails_helper.rb

```diff
@@ -32,6 +45,8 @@
   # config.mock_with :flexmock
   # config.mock_with :rr
 
+  config.include WaitForAjax, type: :feature
+
```

Comment:
I've had good success triggering the error by running rspec with a set seed:
`bundle exec rspec ./spec/features/user_updates_password_spec.rb --seed 15515`

I also attempted to install a global `after(:each)` by adding:

```ruby
  config.after(:each, type: :feature, js: true) {
    puts 'AFTER EACH'
    wait_for_ajax
  }
```

The blog is executed, but the error still persists.  However when I add the `wait_for_ajax` call to the the spec directly it seems to suppress the error.